AirBench/Models/Bench.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AirBench.Models
+{
+    public class Bench
+    {
+        public int Id { get; set; }
+        [Required]
+        public decimal Rating { get; set; }
+        [Required, MaxLength(255)]
+        public string Description { get; set; }
+        [Required]
+        public int Seats { get; set; }
+        [Required]
+        public decimal Latitude { get; set; }
+        [Required]
+        public decimal Longitude { get; set; }
```

Comment:
By default, non-nullable value types won't allow nulls in the database, so adding the `Required` attribute is redundant.